.github/workflows/ipr.yml

```
@@ -12,6 +12,7 @@ jobs:
   build:
     name: 'check IPR form'
     runs-on: ubuntu-latest
+    if: ${{ github.repository == 'tc39/ecma262' }}
```

```suggestion
    if: ${{ secrets.GH_IPR_TOKEN != '' && secrets.GOOGLE_API_KEY != '' }}
```
i think this might do it?

---

Doesn't this increase the risk of silent failure if e.g. a secret is accidentally removed or renamed?

---

yes, but that's not a meaningful risk because they're required branch protections - we'll discover it pretty quickly.

---

If you say so. Accepted.